.github/workflows/manual-publish.yml

```diff
@@ -0,0 +1,52 @@
+name: Publish Images and Artifacts
+on:
+  workflow_dispatch:
+    inputs:
+      dry_run:
+        description: 'Skip publishing to DockerHub'
+        type: boolean
+        required: false
+        default: true
+      tag:
+        description: 'Tag to upload binary artifacts to.'
+        type: string
+        required: true
+
+jobs:
+  release-ldcli:
+    permissions:
+      id-token: write # Needed to obtain Docker tokens
+      contents: write # Needed to upload release artifacts
+    if: ${{ needs.release-please.outputs.release_created == 'true' }}
+    runs-on: ubuntu-latest
+    outputs: 
+      hashes: ${{ steps.publish.outputs.hashes }}
+    steps:
+      - uses: actions/checkout@v4
+        name: Checkout
+        with:
+          fetch-depth: 0
+          
+      - uses: launchdarkly/gh-actions/actions/release-secrets@release-secrets-v1.0.1
+        name: 'Get Docker token'
+        with:
+          aws_assume_role: ${{ vars.AWS_ROLE_ARN }}
+          ssm_parameter_pairs: '/global/services/docker/public/username = DOCKER_USERNAME, /global/services/docker/public/token = DOCKER_TOKEN'
+          
+      - uses: ./.github/actions/publish
+        with:
```

Comment:
I think we need to add an `id: publish` here so that `${{ steps.publish.outputs.hashes}}` above works

Reply:
Added!